package.json

```diff
@@ -1,12 +1,13 @@
 {
   "dependencies": {
-    "@rails/webpacker": "^5.1.1",
-    "datatables.net-bs4": "^1.10.19",
-    "datatables.net-buttons-bs4": "^1.5.6",
-    "datatables.net-fixedcolumns-bs4": "^3.2.6",
-    "datatables.net-fixedheader-bs4": "^3.1.5",
-    "datatables.net-responsive-bs4": "^2.2.3",
-    "datatables.net-rowgroup-bs4": "^1.1.0",
```

Comment:
Lots of updates. Webpacker in particular is nice, as it drops the node-sass dependency which regularly gives me pain.